src/test/resources/features/03-login/login-test.feature

```diff
@@ -0,0 +1,11 @@
+Feature: User Login check
+  I want to be able to see if a valid user login correctly
+
+  Scenario: Open the home page fill in credentials and login
+    Given User opens the website at "https://openmrs-spa.org/openmrs/spa"
```

Comment:
you don't need the url in your steps since it can change. https://automationpanda.com/2017/01/30/bdd-101-writing-good-gherkin/

Reply:
Alright... I'll add a base Url

Reply:
Thank you